flyteplugins/go/tasks/plugins/k8s/kfoperators/common/common_operator.go

```diff
@@ -6,7 +6,7 @@ import (
 	"sort"
 	"time"
 
-	commonOp "github.com/kubeflow/common/pkg/apis/common/v1"
+	kubeflowv1 "github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v1"
 	v1 "k8s.io/api/core/v1"
```

Comment:
`github.com/kubeflow/common` has been integrated into `github.com/kubeflow/training-operator`. This leads to a lot of `commonOp` -> `kubeflowv1` changes below.